riscv/tests/riscv.rs

```diff
@@ -185,7 +185,8 @@ fn memfuncs() {
 #[ignore = "Too slow"]
 fn keccak() {
     let case = "keccak";
-    verify_riscv_crate(case, Default::default(), false);
+    let options = CompilerOptions::new_gl().with_keccak();
+    verify_riscv_crate_gl_with_options(case, Default::default(), options, false);
```

Comment:
I think this has to be reverted so software keccak is still tested, and need to add a new test for the acc version